src/app/globals.css

```diff
@@ -4,7 +4,13 @@
   box-sizing: border-box;
 }
 
+@font-face {
+  font-family: poppins;
+  src: url(src/assets/Poppins-Medium.ttf);
+}
+
 :root {
+  --header-font: 'poppins', sans-serif;
```

Comment:
I would've expected a more general variable name, but this issue will be obsolete once we merge #8